pagopa/pn-frontend-e2e
# PN-10235-e-2-e-destinatario-sezione-notifiche-inserimento-formato-data-errato

## Short description

Implemented PN-10235 scenario B - invalid date format 

## List of changes proposed in this pull request

Implemented scenario B PN-10235
added DestinatarioPage class for methods in common between PG and PF

## How to test

In destinatario notifiche section insert invalid format date (like 01/01/1111) and verify the field is turning red
